src/main/java/org/jenkinsci/test/acceptance/docker/DockerImage.java

```diff
@@ -12,10 +13,14 @@
  * @author Kohsuke Kawaguchi
  */
 public class DockerImage {
+
+    public static final String DEFAULT_DOCKER_HOST = "127.0.0.1";
     public final String tag;
+    DockerHostResolver dockerHostResolver;
 
     public DockerImage(String tag) {
         this.tag = tag;
+        dockerHostResolver = new DockerHostResolver();
```

Comment:
Well one way of keeping it simple is to follow the conventions in the rest of the system, suddenly switching conventions in the middle of it all can confuse casual observers of the code. But I have no strong opinions of it in this particular case.